include/xtensor/xoperation.hpp

```diff
@@ -733,7 +733,7 @@ namespace xt
      * @brief return vector of indices where arr is not zero
      *
      * @param arr input array
-     * @return vector of index_types where arr is not equal to zero
+     * @return vector of index_types where arr is not equal to zero (use `xt::from_indices` to convert)
```

Comment:
maybe we can also use `@sa from_indices` here so that it appears as a `see also: xt::from_indices...`.

Reply:
Like this?